### DISCUSSION TITLE
标题过长，导致封面占了两页

### DISCUSSION THREAD

**Discussion by YifengZhaoo** [TheNetAdmin/zjuthesis#412] [General]
我的中文标题和英文标题都比较长，按照 https://github.com/TheNetAdmin/zjuthesis/discussions/331 进行了修改，但导致封面部分占了两页

尝试调整字体大小解决，未成功

**Comment by shuyangwei** [comment]
请问你的问题解决了吗？

**Reply by xfz329** [reply]
适当的调整下\page\undergraduate\final\cover.tex 中 \vskip 相关语句的后面数值，适当的减小该数值，同时略微缩小字体大小，多试几次就可以了。这个过程只能hard code 自己调了

**Reply by shuyangwei** [reply]
谢谢谢谢🙏，我按照你的方式试一试。叨扰了！

**Reply by xfz329** [reply]
硕士的话需要修改这个文件 \page\graduate\cover-chn.tex，该目录下的其他几个cover相关的文件可能都需要修改下

**Comment by xfz329** [comment]
适当的调整下\page\undergraduate\final\cover.tex 中 \vskip 相关语句的后面数值，适当的减小该数值，同时略微缩小字体大小，多试几次就可以了。这个过程只能hard code 自己调了

**Comment by Mingyi1231** [comment]
![image](https://github.com/user-attachments/assets/928707f3-ada0-4b77-a5d9-769f0cdb4261)
cover_chn.tex文件里面改了这部分就好啦分享一下~
